presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -789,4 +795,9 @@ default ConnectorTableProperties getTableProperties(ConnectorSession session, Co
     {
         return Optional.empty();
     }
+
+    default boolean supportsMissingColumnsOnInsert()
```

Comment:
Please  add javadoc

Reply:
the javadoc says same thing as `createTableWithDefaultColumns`.  I would say something this is not clear from the code. Maybe something about `default` values.